master/internal/trial.go

```diff
@@ -1030,8 +1030,13 @@ func classifyStatus(state terminatedContainerWithState) aproto.ContainerStopped
 }
 
 func (t *trial) reset() error {
-	step := t.sequencer.RollBackSequencer()
-	if err := t.db.RollBackTrial(t.id, step); err != nil {
+	t.sequencer.RollBackSequencer()
+	w, err := t.sequencer.Workload()
```

Comment:
maybe just open up a CurrentBatch() on the sequencer?

Reply:
I guess when I was thinking about this, I was trying to make sure that called `t.sequencer.Workload()` is always valid when we'll be resetting.. I think yeah, for sure. We should never reset then have nothing to do.